src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -531,7 +533,8 @@ class SlickDataAccess(databaseConfig: Config) extends DataAccess {
             key.index.fromIndex,
             IoOutput,
             wdlValue.wdlType.toWdlString,
-            Option(wdlValueToDbValue(wdlValue).toClob))
+            Option(wdlValueToDbValue(wdlValue).toClob),
+            None) // TODO: Use the symbol hash (then update SlickDataAccessSpec:603)!
```

Comment:
Oh yeah, I remember now - this was a nightmare because of the fileHasher implicit arg